plugins/ingest-attachment/src/main/java/org/elasticsearch/ingest/attachment/AttachmentProcessor.java

```diff
@@ -127,7 +127,7 @@ public void execute(IngestDocument ingestDocument) {
                 }
                 additionalFields.put(Property.CONTENT_LENGTH.toLowerCase(), length);
             }
-        } catch (Exception e) {
+        } catch (Throwable e) {
```

Comment:
Let's not do this. We worked incredibly hard to remove it from the codebase, see #19231. It just hides bugs, and swallows truly fatal errors.